### PR TITLE
feat: add versioning

### PR DESCRIPTION
**Pull request by sarahzdev**
**Summary**
- Made of a copy of current docs into 2.0.0
- Added toggle in navbar to switch between v2 and v3
- Set up redirect to v3

### CODE REVIEW

File: docs/SDK/intro.mdx [hunk 1]
```diff
@@ -2,10 +2,10 @@
  sidebar_position: 1
 --- 
 
-import LitActionsSection from '../../src/components/SDKCardSections/Actions';
-import AccessControlSection from '../../src/components/SDKCardSections/AccessControl';
-import ResourcesSection from '../../src/components/SDKCardSections/Resources';
-import QuickStartSection from '../../src/components/SDKCardSections/QuickStart';
+import LitActionsSection from '@site/src/components/SDKCardSections/Actions';
```
Comment (hwrdtm) [inline docs/SDK/intro.mdx:5]: What does `@site` do?

File: docusaurus.config.js [hunk 2]
```diff
@@ -22,17 +22,27 @@ const config = {
...
+          breadcrumbs: false,
+          lastVersion: "current",
           sidebarPath: require.resolve("./sidebars.js"),
           // Please change this to your repo.
           editUrl: "https://github.com/LIT-Protocol/docs/edit/main/website/",
           routeBasePath: "/",
+          versions: {
+            '2.0': {
+              badge: false,
+              label: 'v2.x.x',
```
Comment (hwrdtm) [inline docusaurus.config.js:34]: maybe this should be 
```suggestion
              label: '< v2.x.x',
```
to capture versions `0.x.x` and `1.x.x`

File: sidebars.js [hunk 3]
```diff
@@ -26,19 +26,19 @@ const sidebars = {
     },
   ],
    */
-  learningLab: [
-    {
-      type: 'autogenerated',
-      dirName: 'learningLab',
-    },
-  ],
+  // learningLab: [
```
Comment (hwrdtm) [inline sidebars.js:29]: should we just remove this?

File: versioned_docs/version-2.0/LitActions/additionalExamples/generatingSessionKey.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      derivedVia: "web3.eth.personal.sign",
+      signedMessage: toSign,
+      address: wallet.address,
+    };
+
+    return authSig;
+  };
+
+  let sessionSigs = await litNodeClient.getSessionSigs({
+    resources: ["litAction://*"],
```
Comment (hwrdtm) [inline versioned_docs/version-2.0/LitActions/additionalExamples/generatingSessionKey.md:69]: Marking this as something we need to change in a future PR - our docs in this section are outdated.

**Review by hwrdtm — COMMENTED** [review]
If we merge this PR now we should probably mark the latest version as 2, since we have yet to do the version bump to 3.x.x on the SDK.

File: docusaurus.config.js (hunk 2 above)
Comment (sarahzdev) [reply docusaurus.config.js:34]: @hwrdtm the content changes between js-sdk v3 and v2.

File: docs/SDK/intro.mdx (hunk 1 above)
Comment (sarahzdev) [reply docs/SDK/intro.mdx:5]: @hwrdtm [docusaurus recommends(https://docusaurus.io/docs/versioning#use-absolute-import-within-the-docs) using absolute import when versioning

File: docusaurus.config.js (hunk 2 above)
Comment (hwrdtm) [reply docusaurus.config.js:34]: right - just wondering which version would the users using `1.x.x` know to refer to?

Comment (sarahzdev) [reply docusaurus.config.js:34]: we launched the ts sdk with 2.x.x^ and removed lit-js-sdk references. as a lesson learned with the lit-js-sdk deprecation, we should keep old content and introduce doc versioning moving forward

**Review by hwrdtm — APPROVED** [review]

**Comment by hwrdtm** [comment]
Need to fix CI. Haven't been able to reproduce locally, so I'm trying to get the node and yarn versions on Netlify to be the same as my local.

**Pull request merged**